justfile

```diff
@@ -219,8 +207,12 @@ prepare-sqlite: install-sqlx
 
 # Install SQLX cli if not already installed.
 [private]
-install-sqlx:
-    @if ! command -v cargo-sqlx &> /dev/null; then \
-        echo "SQLX cargo plugin could not be found. Installing..." ;\
-        cargo install sqlx-cli --no-default-features --features sqlite,native-tls ;\
+install-sqlx: (cargo-install "cargo-sqlx" "sqlx-cli" "--no-default-features" "--features" "sqlite,native-tls")
+
+# Check if a certain Cargo command is installed, and install it if needed
+[private]
+cargo-install $COMMAND $INSTALL_CMD="" *ARGS="":
```

Comment:
neat!

Reply:
yeah, i am liking justfiles more and more every day :)